react-components/src/components/Architecture/CommandButton.tsx

```diff
@@ -20,7 +21,13 @@ export const CommandButtons = ({
   return (
     <>
       {commands.map(
-        (command, index): ReactElement => addCommandButton(command, isHorizontal, index)
+        (command, index): ReactElement => (
+          <CommandButtonWrapper
+            command={command}
+            isHorizontal={isHorizontal}
+            key={getKey(command, -index)}
```

Comment:
Why `-index`?

Reply:
Turns out commands have a numerical key, and it's a running number starting at 1. So chances of collision is pretty high if the index is used as-is